pages/common/mycli.md

```diff
@@ -0,0 +1,15 @@
+# mycli
+
+> A command line client for MySQL that can do auto-completion and syntax highlighting.
+
+- Connect local database with port 3306 and current user's username:
+
+`mycli {{database_name}}`
+
+- Connect to a database, user will be prompted for a password:
```

Comment:
This is a [comma splice](https://en.wikipedia.org/wiki/Comma_splice). We can avoid it with parenthesis:

```suggestion
- Connect to a database (user will be prompted for a password):
```

Or, alternatively, we could use a semicolon instead of a comma.

Reply:
Interesting. I didn't know that grammatical construct had a name.  A semi-colon does work here - but I agree that it would sound better with brackets, yeah.